ext/build_defaults.yaml

```diff
@@ -85,6 +86,8 @@ platform_repos:
     repo_location: repos/sles/12/**/x86_64
   - name: sles-12-ppc64le
     repo_location: repos/sles/12/**/ppc64le
+  - name: sles-15-x86_64
+    repo_location: repos/sles/15/**/x86_64
```

Comment:
Does `rpm_targets` need to be updated below, too?

Reply:
I checked with RelEng, and it sounds like rpm_targets and deb_targets only list PE master platforms, and these vars are used in compose and promotion. So I wouldn't want to add sles-15 to that.